src/styles/main.scss

```diff
@@ -396,3 +380,26 @@ ul.resources-nav-tabs button.active {
 .icon-lg {
   font-size: 20px;
 }
+
```

Comment:
Yes. I didn't want to change the background colors now because they would have been the same as the right panels. Seems like a single ticket to change all background colors makes sense.

Reply:
Done, ticket https://github.com/LD4P/sinopia_editor/issues/2408.